### PR TITLE
add contributor && fix title issues

### PR DESCRIPTION
**Pull request by rajagopal4890**
Forgot to add myself as the contributor. Also, the h4 title for the file names is not working in the website. I had put the file names as comment under code snippet.

### CODE REVIEW

File: content/guides/production-build.md [hunk 1]
```diff
@@ -71,12 +74,14 @@ module.exports = {
...
 ```
-##Configuring webpack for multiple environments
+## Configuring webpack for multiple environments
 
 When we do have multiple configurations in mind for different environments, the easiest way is to write seperate js files for 
 each environment. For example:
-####config/dev.js
+
 ```js
+//config/dev.js
```
Comment (bebraw) [inline content/guides/production-build.md]: You could handle the filenames likethis:

**config/dev.js**

```js
...
```

So bold on a separate line.

Apart from that this is good.

**Comment by bebraw** [comment]
Looks like there's a conflict (other PR got merged). Could you rebase and force push the PR?

File: content/guides/production-build.md (hunk 1 above)
Comment (rajagopal4890) [inline content/guides/production-build.md]: Thanks. made the changes. Hosted the same on my localhost as well. title looks good.

**Comment by rajagopal4890** [comment]
Yup. Will do and get back.

**Comment by rajagopal4890** [comment]
@bebraw, I did the merge.

**Comment by bebraw** [comment]
Hi,

It caught some unnecessary commits. I think you can eliminate those like this:

```bash
git rebase master
git push --force
```

The first operation will get this in sync. You may have to resolve some conflicts. Force push should be fine since this is a feature branch (only you use it).

**Pull request closed without merging**